crates/router/src/core/payments/transformers.rs

```diff
@@ -1809,6 +1816,22 @@ impl<F: Clone> TryFrom<PaymentAdditionalData<'_, F>> for types::PaymentsAuthoriz
         ));
 
         // payment_method_data is not required during recurring mandate payment, in such case keep default PaymentMethodData as MandatePayment
+        let additional_payment_method_data = if payment_data.mandate_id.is_some() {
```

Comment:
Is this comment applicable for below code too?

Reply:
Not needed